backend/src/main/java/gov/cdc/usds/simplereport/db/repository/TestEventRepository.java

```diff
@@ -23,6 +23,14 @@
 
   public TestEvent findFirst1ByPatientOrderByCreatedAtDesc(Person p);
 
+  @Query(
+      value =
+          " SELECT DISTINCT ON (patient_id) *, COALESCE(date_tested_backdate, created_at) test_date FROM {h-schema}test_event"
```

Comment:
I'm guessing the answer to this is "no" since the tests are passing, but don't you need an `as` here? E.g., `COALESCE(...) AS test_date`

Reply:
The `AS` keyword is optional when aliasing columns in a `SELECT`, but I'll add it in 😄